### ISSUE TITLE
Abandon claim cause Dicefurniture furniture disappear

### ISSUE THREAD

**Issue by hkkongou** [GriefPrevention/GriefPrevention#1171]
### What happened:
_What you saw_
when removing the a claim using /abandomclaim, dicefurniture inside claim will disappear
https://user-images.githubusercontent.com/61608872/102840695-60f0ff80-443e-11eb-8351-5369b851cb55.mp4

### What was expected:
furniture keep in the same place even the claim is removed

### Steps to reproduce:
__This may include a build schematic, a video, or detailed instructions to help reconstruct the issue__
1. make a claim
2. put furniture inside claim
3. abandonclaim

### Server and GriefPrevention version:
___This can be found by running `/version` and `/version GriefPrevention` on your server.___
Paste of `/version`: 
Paste of `/version GriefPrevention`: 
![image](https://user-images.githubusercontent.com/61608872/102840848-ad3c3f80-443e-11eb-9f12-92e512037c97.png)

![image](https://user-images.githubusercontent.com/61608872/102840830-a6adc800-443e-11eb-9ed3-7a16afe607c9.png)

### Stack trace/error or server log
no error log 
```
Paste entire stack trace right here inside these backticks.

```
Or, paste the server.log at gist.github.com and paste the link here: 
https://drive.google.com/drive/folders/1uHzLL84Q_tXLCtAeRKha6QEsDw_UTmca?usp=sharing
### GriefPrevention config.yml

```yml

GriefPrevention:
  SeaLevelOverrides:
    world: -1
    world_nether: -1
    world_the_end: -1
    world_tutorial: -1
    world_halloween: -1
    world_resource: -1
    world_dungeon: -1
    event: -1
    world_main: -1
  Claims:
    Mode:
      world_resource: Disabled
      world_nether: Disabled
      world_halloween: Survival
      world_the_end: Disabled
      world_tutorial: Survival
      event: Survival
      world_main: Survival
      world: Survival
      world_dungeon: Disabled
    PreventGlobalMonsterEggs: false
    PreventTheft: true
    ProtectCreatures: true
    PreventButtonsSwitches: true
    LockWoodenDoors: true
    LockTrapDoors: true
    LockFenceGates: true
    EnderPearlsRequireAccessTrust: true
    RaidTriggersRequireBuildTrust: true
    ProtectHorses: true
    ProtectDonkeys: true
    ProtectLlamas: true
    InitialBlocks: 2500
    Claim Blocks Accrued Per Hour:
      Default: 300
    Max Accrued Claim Blocks:
      Default: 50000
    Accrued Idle Threshold: 0
    AccruedIdlePercent: 0
    AbandonReturnRatio: 1.0
    AutomaticNewPlayerClaimsRadius: -1
    AutomaticNewPlayerClaimsRadiusMinimum: 0
    ExtendIntoGroundDistance: 10
    MinimumWidth: 5
    MinimumArea: 100
    MaximumDepth: 0
    InvestigationTool: STICK
    ModificationTool: GOLDEN_SHOVEL
    Expiration:
      ChestClaimDays: 0
      UnusedClaimDays: 14
      AllClaims:
        DaysInactive: 60
        ExceptWhenOwnerHasTotalClaimBlocks: 20000
        ExceptWhenOwnerHasBonusClaimBlocks: 5000
      AutomaticNatureRestoration:
        SurvivalWorlds: false
    AllowTrappedInAdminClaims: false
    MaximumNumberOfClaimsPerPlayer: 0
    CreationRequiresWorldGuardBuildPermission: true
    VillagerTradingRequiresPermission: true
    CommandsRequiringAccessTrust: /sethome
    DeliverManuals: false
    ManualDeliveryDelaySeconds: 30
    RavagersBreakBlocks: false
    FireSpreadsInClaims: false
    FireDamagesInClaims: false
    LecternReadingRequiresAccessTrust: true
  Spam:
    Enabled: false
    LoginCooldownSeconds: 5
    LoginLogoutNotificationsPerMinute: 5
    ChatSlashCommands: /me;/global;/local
    WhisperSlashCommands: /tell;/pm;/r;/whisper;/msg
    WarningMessage: Please reduce your noise level.  Spammers will be banned.
    BanOffenders: true
    BanMessage: Banned for spam.
    AllowedIpAddresses: 1.2.3.4; 5.6.7.8
    DeathMessageCooldownSeconds: 120
    Logout Message Delay In Seconds: 0
  PvP:
    RulesEnabledInWorld:
      world: false
      world_nether: false
      world_the_end: false
      world_tutorial: false
      world_halloween: true
      world_resource: false
      world_dungeon: false
      event: false
      world_main: false
    ProtectFreshSpawns: false
    PunishLogout: false
    CombatTimeoutSeconds: 0
    AllowCombatItemDrop: true
    BlockedSlashCommands: /home;/vanish;/spawn;/tpa
    ProtectPlayersInLandClaims:
      PlayerOwnedClaims: false
      AdministrativeClaims: false
      AdministrativeSubdivisions: false
    AllowLavaDumpingNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    AllowFlintAndSteelNearOtherPlayers:
      PvPWorlds: false
      NonPvPWorlds: false
    ProtectPetsOutsideLandClaims: false
  Economy:
    ClaimBlocksMaxBonus: 0
    ClaimBlocksPurchaseCost: 0.0
    ClaimBlocksSellValue: 0.0
  ProtectItemsDroppedOnDeath:
    PvPWorlds: false
    NonPvPWorlds: false
  BlockLandClaimExplosions: true
  BlockSurfaceCreeperExplosions: false
  BlockSurfaceOtherExplosions: false
  LimitSkyTrees: false
  LimitTreeGrowth: false
  PistonMovement: IGNORED
  PistonExplosionSound: true
  FireSpreads: false
  FireDestroys: false
  AdminsGetWhispers: false
  AdminsGetSignNotifications: false
  SmartBan: false
  Mute New Players Using Banned Words: false
  MaxPlayersPerIpAddress: 2
  Siege:
    Worlds: []
    BreakableBlocks:
    - DIRT
    - GRASS_BLOCK
    - GRASS
    - FERN
    - DEAD_BUSH
    - COBBLESTONE
    - GRAVEL
    - SAND
    - GLASS
    - GLASS_PANE
    - OAK_PLANKS
    - SPRUCE_PLANKS
    - BIRCH_PLANKS
    - JUNGLE_PLANKS
    - ACACIA_PLANKS
    - DARK_OAK_PLANKS
    - WHITE_WOOL
    - ORANGE_WOOL
    - MAGENTA_WOOL
    - LIGHT_BLUE_WOOL
    - YELLOW_WOOL
    - LIME_WOOL
    - PINK_WOOL
    - GRAY_WOOL
    - LIGHT_GRAY_WOOL
    - CYAN_WOOL
    - PURPLE_WOOL
    - BLUE_WOOL
    - BROWN_WOOL
    - GREEN_WOOL
    - RED_WOOL
    - BLACK_WOOL
    - SNOW
    DoorsOpenDelayInSeconds: 300
    CooldownEndInMinutes: 60
  EndermenMoveBlocks: false
  SilverfishBreakBlocks: false
  CreaturesTrampleCrops: false
  RabbitsEatCrops: true
  HardModeZombiesBreakDoors: false
  Database:
    URL: ''
    UserName: ''
    Password: ''
  UseBanCommand: true
  BanCommandPattern: ban %name% %reason%
  Advanced:
    fixNegativeClaimblockAmounts: true
    ClaimExpirationCheckRate: 60
    OfflinePlayer_cache_days: 90
  Abridged Logs:
    Days To Keep: 7
    Included Entry Types:
      Social Activity: true
      Suspicious Activity: true
      Administrative Activity: false
      Debug: false
      Muted Chat Messages: false

Paste the config.yml right here inside these backticks. Please do not use a paste service for this.

```

### Plugin list (if applicable):
![image](https://user-images.githubusercontent.com/61608872/102841021-13c15d80-443f-11eb-820a-61e1d441a06e.png)

### Debugging steps attempted

- [✅] I attempted running only GriefPrevention on the server
  - [❎] Issue still occurs even if GriefPrevention is the only plugin running
- [✅] I attempted testing for the issue on a new server
  - [✅] Issue still occurs even after testing on a new server
- [❎] I attempted testing for the issue with the vanilla client (no mods)
  - [❎] Issue still occurs even after testing with the vanilla Minecraft client

**Comment by RoboMWM** [comment]
config needed. Also, don't use emojis for the checkboxes, just fill out with an x so that the github flavored markdown applies the checkbox formatting.

**Comment by RoboMWM** [comment]
without config it is unknown, but you either have creative claims enabled or using cleanup in survival. Please see documentation regarding entity cleanup.

**Comment by hkkongou** [comment]
The config is already uploaded in Google drive 16hours before on the above link given 16 hours before

**Comment by Jikoo** [comment]
> Paste the config.yml right here inside these backticks. Please do not use a paste service for this.

From the template.

**Comment by RoboMWM** [comment]
also, it's in the wrong section, so that's why I missed it.

**Comment by hkkongou** [comment]
sorry i don't know english well, but is fine now should be

**Comment by RoboMWM** [comment]
hmm, wonder if the Dicefurniture plugin is doing something then, since I don't see any creative worlds here nor is restore enabled.

**Comment by Jikoo** [comment]
Appears to be a DiceFurniture feature. The source is really messy, so it's hard to follow exactly what's going on without being familiar with the project, but it clearly has a [protection manager](https://github.com/Ste3et/FurnitureLib/blob/17d23841d04081e23928bfec7b8b401113847e18/src/de/Ste3et_C0st/FurnitureLib/main/Protection/ProtectionManager.java) system involving ProtectionLib. Check your ProtectionLib/DiceFurniture/FurnitureLib settings, and if you can't figure it out contact them. This is not a problem that can be solved on GP's end.
